.github/workflows/stable.yml

```diff
@@ -184,6 +184,32 @@ jobs:
           command: test
           args: --target=i686-pc-windows-msvc --verbose
 
+  quiche_linux_arm:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: ["aarch64-unknown-linux-gnu", "armv7-unknown-linux-gnueabihf"]
```

Comment:
The difference is `quiche_x86` runs on a native host so it will build much faster (but no test), where arm build is using docker + qemu for each arch so it will be slower, but we can run `cargo test`.

Reply:
Added f5110e8 for moving x86 target to `multiarch` and rebased.

Reply:
Looks like Window target is slowest (and x86 need a docker container only, not running in a emulator) so it should be ok.